src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -600,6 +607,33 @@ public boolean canProvision(@NonNull Cloud.CloudState state) {
     public PodTemplate getTemplate(@CheckForNull Label label) {
         return PodTemplateUtils.getTemplateByLabel(label, getAllTemplates());
     }
+    
+    @CheckForNull
+    public PodTemplate getTemplate(@NonNull String id) {
```

Comment:
is it used somewhere in jelly?

Reply:
To edit a template https://github.com/jenkinsci/kubernetes-plugin/pull/1443/files#diff-dd8d64b0b50b10c779638c206b8538d1c701f2498efcf0f1ea22050ad309d588R37